app/api/schema/events.py

```diff
@@ -272,6 +272,14 @@ def validate_timezone(self, data, original_data):
                                         related_view_kwargs={'event_id': '<id>'},
                                         schema='StripeAuthorizationSchema',
                                         type_='stripe-authorization')
+    order_statistics_event = Relationship(attribute='order_statistics_event',
```

Comment:
@iamareebjamal - Please guide me here, this is not working when I hit with include. Am I doing it correct? or their is some another way

Reply:
See how it is done for other stuff. It is related to the urls, related and self. See the flask rest jsonapi docs. It's not obvious how to make it work where the data is unrelated to foreign key of the item. There is a guide in the docs on how to do it. I haven't done in previously. And docs are not great and very few people use the library and it is kinda abandoned